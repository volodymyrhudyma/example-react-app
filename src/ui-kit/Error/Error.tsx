import React from 'react';
import { Wrapper } from './styled/Error.styled';

interface IErrorProps {
  message: string;
}

const Error = ({ message }: IErrorProps) => <Wrapper>{message}</Wrapper>;

export default Error;
