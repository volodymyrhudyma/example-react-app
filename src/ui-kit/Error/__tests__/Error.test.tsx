import React from 'react';
import { shallow } from 'enzyme';
import Error from '../Error';

describe('Error component', () => {
  it('should render correctly', () => {
    const component = shallow(<Error message="Test" />);
    expect(component).toMatchSnapshot();
  });
});
