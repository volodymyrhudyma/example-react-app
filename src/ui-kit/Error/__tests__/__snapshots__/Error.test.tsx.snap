// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render correctly 1`] = `
<styled.div>
  Test
</styled.div>
`;
